db/migrate/20241011054348_fix_unique_constraint_on_problem_check_trackers.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class FixUniqueConstraintOnProblemCheckTrackers < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
```

Comment:
I don't think you can disable the transaction here. The sequence of the migration is something like this:

1. Drop old unique index
2. Clean up duplicated records
3. Create new unique index.  

Since transaction is disabled, there is a somewhat small chance that an existing duplicated record can be inserted between step 2 and 3. If that happens, this migration will fail. 

Since the `problem_check_trackers` table is likely to be small, I think we can just run the migration within a transaction and just drop `algorith: :concurrently`?

Reply:
Actually that was my original approach, so I'm all for it. 👍